modules/auxiliary/scanner/msmail/shared/src/msmail/msmail.go

```diff
@@ -0,0 +1,102 @@
+package msmail
+
+import (
+	"crypto/tls"
+	"encoding/base64"
+	"metasploit/module"
+	"net/http"
+	"strings"
+	"time"
+)
+
+func WebRequestCodeResponse(URI string) int {
```

Comment:
I'll try and loop back on a patch on this, but realistically this should probably return an [`http.Response`](https://golang.org/pkg/net/http/#Response) and `error` instead of an integer.